index.js

```diff
@@ -19,6 +20,7 @@ var randombytes = require('randombytes')
 var speedometer = require('speedometer')
 var zeroFill = require('zero-fill')
 
+var dhtPersist = require('./lib/dhtpersist') // browser exclude
```

Comment:
It says browser exclude, but it's not actually excluded in the browser field of package.json?

like this 

``` json
"browser": {
  "./lib/dhtpersist": false
}
```

Reply:
Misunderstood what that was about. I was thinking it was some control message to the bundler. I've updated.